### PR TITLE
feat: Improve hint icon alignment across the application

### LINKED ISSUE #84: Improve hint icon alignment across the application
Improve hint icon alignment across the application, some of them are not aligned with the content.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #84 

## Demo

https://github.com/aeternity/aescan/assets/15363559/81db6d2c-d2ac-452e-85d5-03788d39e335

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] unify icon usage
- [x] decrease size of icon

### CODE REVIEW

File: src/components/HintIcon.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <app-icon
     name="question"
-    :size="19"
+    :size="16"
```
Comment (michele-franchi) [inline src/components/HintIcon.vue:4]: If It does not break the alignment I would give a size of at least `18`, now It looks a bit too small.

**Review by michele-franchi — COMMENTED** [review]
I suggested a different size of the icon.

File: src/components/HintIcon.vue (hunk 1 above)
Comment (lukeromanowicz) [reply src/components/HintIcon.vue:4]: I think we might need different sizes based on where it's used. For example, a hint icon next to h1 should probably be bigger than in table headers.

Comment (michele-franchi) [reply src/components/HintIcon.vue:4]: That's another improvement but for now I would simply make it a little bit bigger.

Comment (janmichek) [reply src/components/HintIcon.vue]: Alright, I extended it to 17.
I took the inspiration from some other explorers where the icon is even smaller
image

image

**Review by michele-franchi — APPROVED** [review]
Looks good to me

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**